msal/src/main/java/com/microsoft/identity/client/AuthenticationResult.java

```diff
@@ -108,4 +117,26 @@ public IAccount getAccount() {
     public String[] getScope() {
         return mAccessToken.getTarget().split("\\s");
     }
+
+    @Nullable
+    @Override
+    public UUID getCorrelationId() {
+        return mCorrelationId;
+    }
+
+    private UUID sanitizeCorrelationId(@Nullable final String correlationId) {
+        final String methodName = "sanitizeCorrelationId";
+
+        if (TextUtils.isEmpty(correlationId)) {
+            Logger.warn(TAG + methodName, "Correlation was empty, returning null.");
```

Comment:
nit: missing id

Reply:
Addressed here: https://github.com/AzureAD/microsoft-authentication-library-for-android/pull/1063/commits/6a71d1d9fc3d3329f4b608dd3702a39ff7099b62